planningDocument.md

```diff
@@ -0,0 +1,82 @@
+# Site Plan - dms.com
+
+## Overview
+
+*dms.com* is a website for the business 'Diversified Manufacturing and Services' (DMS). The goal of this site is to showcase the company, detail the company's products and services, and provide methods of contacting the business. The site will be a professional platform to increase the amount of requests to the business for their specific products and services.
+
+This site will target prospective clients that are looking for an electronics manufacturer. Today it is expected that a legitimate business have a professional-looking website to inform the client and bridge their communication. Therefore, it is necessary that DMS provide an online presence to provide information about the business and facilitate client communication.
+
+The site will be primarily static and built from scratch, custom to the professional environment DMS wishes to project. The content will span several pages and cover a general description of the company, specific manufacturing and design services, and methods of contacting and requesting quotes. The website will focus on responsive design and have a fluid, professional look on all platforms.
+
+## Goals
+
+The primary goals of the site and corresponding objectives are given below.
+
+### Generate Prospective Client Interest
+
+* Increase business related inquiries via email, phone, or in-person. 
```

Comment:
When DMS receives a phone call from a prospective client, it may be interesting to ask how the client was referred. Then you can measure if you reached this goal. (This is not required for this project, I only thought it would be interesting.)